pkg/v3/ocr3_test.go

```diff
@@ -1,4 +1,4 @@
-package v3
+package ocr2keepers
```

Comment:
good catch, this is fixed in another PR thanks to Fergal. You should see it when you rebase.